tests/docker/config/construct-query.sparql

```diff
@@ -267,6 +268,17 @@ CONSTRUCT {
       ) AS ?sTypeIdLabel) .
   } .
 
+  OPTIONAL {
+    ?id nsg:annotation / nsg:hasBody ?curatedAnnotation .
+    BIND(
+      IF(
+         ?curatedAnnotation = <https://neuroshapes.org/Curated>,
+         true,
+         IF(?curatedAnnotation = <https://neuroshapes.org/Unassessed>, false, ?undefined)
```

Comment:
how is this undefined value used? is it like `None`/ `Null`?

Reply:
This should just be a blank node that is never used, this way if the resource doesn't have the curated/unassessed annotation then there will be no `curated` field at all in the ES document; there is also a test below to ensure this is true. Let me know if you know a nicer way to deal with this!